apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.html.ts

```diff
@@ -121,6 +121,28 @@ export function InstructorCourseAdminInstances({
                 </tbody>
               </table>
             </div>
+            ${courseInstances.length === 0
+              ? html`
+                  <div class="my-4">
+                    <p class="text-center font-weight-bold">No course instances found.</p>
+                    <p class="text-center mb-0">
+                      A course instance is a single offering of a course, such as "Fall 2024" or
+                      "Fall 2024, Section M".
+                    </p>
```

Comment:
The text here can appear unbalanced at specific viewports:

<img width="645" alt="Screenshot 2024-12-10 at 09 17 47" src="https://github.com/user-attachments/assets/1ec2df59-69f0-4ddb-a202-1d994f08720f">

There's a newish CSS feature called `text-wrap: balance` that could improve this: https://developer.mozilla.org/en-US/docs/Web/CSS/text-wrap#balance. What do you think about adding that in an inline `style` attribute here?

Reply:
I added the text-wrap: balance styling, so this should be fixed now.